code/datums/components/item_equipped_movement_rustle.dm

```diff
@@ -0,0 +1,80 @@
+/datum/component/item_equipped_movement_rustle
+	var/static/list/default_rustle_sounds = SFX_SUIT_STEP
+
+	var/list/rustle_sounds
+	var/mob/holder
+
+	///what move are we on.
+	var/move_counter = 0
+	///how many moves to take before playing the sound, defaults to 4.
+	var/move_delay = 4
+
+	///volume at which the sound plays, defaults to 20.
+	var/volume = 20
+	///does the sound vary? defaults to true.
+	var/sound_vary = TRUE
+	///extra-range for this component's sound, defaults to -1.
+	var/sound_extra_range = -1
+	///sound exponent for the rustle. Defaults to 5.
+	var/sound_falloff_exponent = 5
+	///when sounds start falling off for the rustle rustle, defaults to SOUND_DEFAULT_FALLOFF_DISTANCE.
+	var/sound_falloff_distance = SOUND_DEFAULT_FALLOFF_DISTANCE
+
+/datum/component/item_equipped_movement_rustle/Initialize(custom_sounds, move_delay_override, volume_override, extrarange, falloff_exponent, falloff_distance)
+	if(!isitem(parent))
+		return COMPONENT_INCOMPATIBLE
+
+	RegisterSignal(parent, COMSIG_ITEM_EQUIPPED, PROC_REF(on_equip))
+	RegisterSignal(parent, COMSIG_ITEM_DROPPED, PROC_REF(on_unequip))
+
+	if(custom_sounds)
+		rustle_sounds = custom_sounds
+	else
+		rustle_sounds = default_rustle_sounds
```

Comment:
you can just set this to a list oft he defaults no reason tomake it a static list (actually introduces potential bugs due to pass byref)

Reply:
I just made it not a list